public_html/wp-content/plugins/wcpt/wcpt-meetup/class-meetup-application.php

```diff
@@ -228,11 +234,9 @@ public function create_post( $data ) {
 		add_post_meta( $post_id, 'Address', $organizer_address );
 		add_post_meta( $post_id, 'Already a meetup', $data['q_already_a_meetup'] );
 		add_post_meta( $post_id, 'Meetup URL', $data['q_existing_meetup_url'] );
-		add_post_meta( $post_id, 'Organizer description', $data['q_describe_yourself'] );
```

Comment:
This & the "Extra Comments" meta are removed here, but still show up in the application in wp-admin. I'm not familiar with the application flow, but maybe this should be filled in with the content from `q_introduction`?

Reply:
IMO those fields are a bit unnecessary because you need to scroll down to the original application nevertheless for seeing all the information in the application. Removing this field makes the flow more similar to WordCamp applications.

Reply:
Like I said, I don't know this flow, by my intuition says this should go one of two ways:

- If these fields are not used for anything, they should be removed from the UI entirely
- If these fields are used, for example if Application Reviewers edit things here, we should still pre-fill them, even if you use the raw data to review.

What do you think?

(We can also open a new issue to better format these answers so that you don't need to look at a raw array dump 🙂 )

Reply:
> If these fields are not used for anything, they should be removed from the UI entirely

I'm not 100% sure if someone uses these fields for something. Usually, these fields shouldn't be edited. I'd agree on removing these fields from UI after some period after the changes to the application form.

> We can also open a new issue to better format these answers so that you don't need to look at a raw array dump

I also thought this and making a PR at some point. For now, we can work with raw array dump like with WordCamp applications - changes to Meetup application form are more urgent as we need to ask more information almost on every application sent.

Reply:
Started another issue to track this idea of improving those metaboxes: https://github.com/WordPress/wordcamp.org/issues/372